contract_sale_invoicing/models/contract.py

```diff
@@ -12,44 +12,78 @@ class ContractContract(models.Model):
         help="If checked include sales with same analytic account to invoice "
         "in contract invoice creation.",
     )
+    invoice_into_contract = fields.Boolean(
```

Comment:
For having the same field prefix to ease recognition:

```suggestion
    invoicing_sales_into_contract = fields.Boolean(
```

Reply:
Done